### PR TITLE
[Chat][Feature] launch chat screen

### PR DESCRIPTION
**Pull request by Mrayyan**
## Purpose

* ...

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

## Pull Request Type
What kind of change does this Pull Request introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Pull Request Checklist

- [ ] Public API changes
- [ ] Verified for cross-platform
- [ ] Synced with iOS, Web team
- [ ] Internal review completed
- [ ] UI Changes
  - [ ] Screen captures included
  - [ ] Tested for Light/Dark mode
  - [ ] Tested for screen rotation
  - [ ] Tested on Tablet and small screen device (5")
  - [ ] Include localization changes
- [ ] Tests
  - [ ] Memory leak analysis performed
  - [ ] Tested on API 21, 26 and latest 
  - [ ] Tested for background mode
  - [ ] Unit Tests Included
  - [ ] UI Automated Tests Included

## How to Test

* Open...
* Click on...

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Comment by iaulakh** [comment]
@Mrayyan  may you please change this PR to :

Only having main Activity that can show button:
No change for demo app
No theming or styling
Just minimal to launch activity with a sample text like "hello"

File: azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/ui/theme/Theme.kt [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    surface = Color.White,
+    onPrimary = Color.White,
+    onSecondary = Color.Black,
+    onBackground = Color.Black,
+    onSurface = Color.Black,
+    */
+)
+
+@Composable
+fun AzurecommunicationuiTheme(
```
Comment (pavelprystinka) [inline azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/ui/theme/Theme.kt:31]: Should AzurecommunicationuiTheme be internal?

File: azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/ui/theme/Type.kt [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package com.azure.android.communication.ui.chat.ui.theme
+
+import androidx.compose.material.Typography
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+
+// Set of Material typography styles to start with
+val Typography = Typography(
```
Comment (pavelprystinka) [inline azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/ui/theme/Type.kt:10]: Please make it internal

File: azure-communication-ui/chat/src/main/res/values/themes.xml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="Theme.Azurecommunicationui" parent="android:Theme.Material.Light.NoActionBar">
```
Comment (pavelprystinka) [inline azure-communication-ui/chat/src/main/res/values/themes.xml:4]: For Calling theme name is "AzureCommunicationUICalling.Theme"
We should use same format for Chat composite as well

File: azure-communication-ui/chat/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="title_activity_chat">ChatActivity</string>
```
Comment (pavelprystinka) [inline azure-communication-ui/chat/src/main/res/values/strings.xml:2]: All resource should start with library name prefix: azure_communication_ui_chat_

File: azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/ui/theme/Shape.kt [hunk 5]
```diff
@@ -0,0 +1,11 @@
+package com.azure.android.communication.ui.chat.ui.theme
+
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Shapes
+import androidx.compose.ui.unit.dp
+
+val Shapes = Shapes(
```
Comment (pavelprystinka) [inline azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/ui/theme/Shape.kt:7]: Please make it internal

File: azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/ui/theme/Color.kt [hunk 6]
```diff
@@ -0,0 +1,8 @@
+package com.azure.android.communication.ui.chat.ui.theme
+
+import androidx.compose.ui.graphics.Color
+
+val Purple200 = Color(0xFFBB86FC)
```
Comment (pavelprystinka) [inline azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/ui/theme/Color.kt:5]: Please make it internal

File: azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/ChatActivity.kt [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import com.azure.android.communication.ui.chat.ui.theme.AzurecommunicationuiTheme
+
+class ChatActivity : ComponentActivity() {
```
Comment (pavelprystinka) [inline azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/ChatActivity.kt:18]: Please make it internal

File: azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/ChatActivity.kt [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+                ) {
+                    Greeting("Android")
+                }
+            }
+        }
+    }
+}
+
+@Composable
+fun Greeting(name: String) {
```
Comment (pavelprystinka) [inline azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/ChatActivity.kt:36]: Please make it internal

File: azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/ChatActivity.kt [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+    Text(text = "Hello $name!")
+}
+
+@Preview(showBackground = true)
+@Composable
+fun DefaultPreview() {
```
Comment (pavelprystinka) [inline azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/ChatActivity.kt:42]: Please make it internal

**Review by pavelprystinka — CHANGES_REQUESTED** [review]

**Comment by iaulakh** [comment]
Closing as per discussion not required

**Pull request closed without merging**